gulp/tasks/styles.js

```diff
@@ -16,8 +17,10 @@ gulp.task('styles', function () {
       sourceMap: 'sass',
       outputStyle: global.isProd ? 'compressed' : 'nested'
     }))
+    .pipe(gulpif(config.logsizes, size({showFiles: true, title:'Pre-autoprefixer'})))
```

Comment:
Could you put the options on separate lines like in the `sass` line right above?